### PR TITLE
Got rid on hard-coded host target in rustwide_builder.rs

### PR DESCRIPTION
**Pull request by Kixiron**
* Now set by the build.rs, its value will be the same as the `TARGET` variable set by cargo
* May have also formatted the build.rs, whoops

### CODE REVIEW

File: src/docbuilder/rustwide_builder.rs [hunk 1]
```diff
@@ -23,8 +23,7 @@ const DEFAULT_RUSTWIDE_WORKSPACE: &str = ".rustwide";
 
 // It is crucial that this be the same as the host that `docs.rs` is being run on.
 // Other values may cause strange and hard-to-debug errors.
-// TODO: use `TARGET` instead? I think `TARGET` is only set for build scripts, though.
-pub(super) const HOST_TARGET: &str = "x86_64-unknown-linux-gnu";
+pub(super) const HOST_TARGET: &str = env!("CRATESFYI_HOST_TARGET"); // Set in build.rs
```
Comment (jyn514) [inline src/docbuilder/rustwide_builder.rs:26]: Is there a way to test this somehow?

File: build.rs [hunk 2]
```diff
@@ -1,57 +1,57 @@
+use git2::Repository;
 use std::env;
-use std::path::Path;
 use std::fs::{self, File};
 use std::io::Write;
-use git2::Repository;
-
+use std::path::Path;
```
Comment (jyn514) [inline build.rs:5]: There's no need to reorder the imports. I like dependency imports being separate from std imports anyway.

File: src/docbuilder/rustwide_builder.rs (hunk 1 above)
Comment (Kixiron) [reply src/docbuilder/rustwide_builder.rs:26]: Not that I know of, tests only have access to the variables that crates themselves do

**Comment by jyn514** [comment]
This looks good but I want to wait to merge until I get a chance to test builds locally.

**Comment by jyn514** [comment]
Looks good to me! Not sure how much it will help builds on windows, but it's at least one more roadblock out of the way.

**Comment by Kixiron** [comment]
Even if it doesn't help with windows builds it's still a good step away from hard-coding, I'm sure this will allow for Mac support

**Pull request merged**